scripts/populate_lambda_ssm_parameters.py

```diff
@@ -102,12 +142,21 @@ def get_admin_user_emails():
         default=None,
         help="Remove a single environment variable in SSM parameters and all deployed lambdas"
     )
+    parser.add_argument("--verify",
+        nargs='*',
```

Comment:
I'd recommend not using nargs and just using a comma-delimited string.  nargs tend to be a bit more ambiguous.

Reply:
kept nargs added support for len(args._action)